catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -620,10 +598,9 @@ private void buildQueue(File inputFile, ArrayBlockingQueue<Metacard> metacardQue
       if (includeContent) {
         try (InputStream data = new FileInputStream(inputFile);
             InputStream signature = new FileInputStream(signatureFile)) {
-          String alias =
-              AccessController.doPrivileged(
-                  (PrivilegedAction<String>)
-                      () -> System.getProperty("org.codice.ddf.system.hostname"));
+          PrivilegedAction<String> action =
```

Comment:
I might be missing it, but was this a finding improvement? Seen this change a couple times in this PR.

Reply:
This was another SolarCloud small finding. It was saying that, we don't need to cast to `(PrivilegedAction<String>)` but when you remove it, the compiler complains that it doesn't know which `doPrivileged()` to call since it has 2 overloaded methods with same amount of params.
e.g.
`doPrivileged(PrivilegedAction<T> action)`
` doPrivileged(PrivilegedExceptionAction<T> action)`